packages/field-plugin/helpers/vite/vite.config.ts

```diff
@@ -17,4 +17,15 @@ export default defineConfig({
       external: ['querystring'],
     },
   },
+  resolve: {
```

Comment:
@eunjae-lee, I may be not seeing something here, but does it make sense to have this alias also for the `vite helper`?

I mean, it seems that the `@storyblok/field-plugin` package is not used in this helper...Am I right?

Reply:
Oh, nice catch! Thanks